### PR TITLE
#168781829 comment notification

### PR DESCRIPTION
**Pull request by urbanishimwe**
#### What does this PR do?
**As a requester or line manager** I should be able to see (in-app) notifications when comments are posted on my request
**So that,** I can determine the next line of action
#### Description of Task to be completed?
- once you create a comment, a notification will be created automatically
in barefoot app
#### How should this be manually tested?
- create a request
- comment on the request
- login the manager of the requester
- get all notifications of the manager
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[story](https://www.pivotaltracker.com/story/show/168781829)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: src/controllers/commentController.js [hunk 1]
```diff
@@ -9,6 +11,15 @@ const CommentsController = {
       userId: req.userData.userId,
       requestId: req.params.requestId,
     });
+    const { lineManager, id } = await UserService.getUser({ id: req.request.UserId });
+    // CREATE NOTIFICATION FOR THE REQUESER OR MANAGER
+    await NotificationService.createNotification({
+      modelName: 'Comments',
+      modelId: newComment.get({ plain: true }).id,
```
Comment (elchroy) [inline src/controllers/commentController.js]: what about just calling `newComment.id`

Comment (urbanishimwe) [reply src/controllers/commentController.js]: that object is a direct instance of `Comments Model`, I can't access `.id` directly!

File: src/utils/echo.util.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+      io.emit(type, data);
+    } else if (connectedClients[receiverId.toString()]) {
+      connectedClients[receiverId.toString()].forEach(element => {
+        io.to(element).emit(type, data);
+      });
+    }
+  }
+};
+
+export default EchoUtil;
```
Comment (lemoissonM) [inline src/utils/echo.util.js]: Awesome! this makes it universal

**Review by lemoissonM — APPROVED** [review]

**Review by Crispy-rw — APPROVED** [review]

File: src/controllers/commentController.js (hunk 1 above)
Comment (urbanishimwe) [reply src/controllers/commentController.js]: I've worked on it!

File: src/utils/echo.util.js (hunk 2 above)
Comment (elchroy) [reply src/utils/echo.util.js]: - Move this method to the `ChatService` and reference it inside the `CommentsController` and `ChatsController`.
- The name `echo` does not seem descriptive enough. Perhaps `sendChatNotification` or something like that.

**Pull request merged**